### ISSUE TITLE
The link for the source in page torch.Tensor.backward is broken.

### ISSUE THREAD

**Issue by qqwqqw689** [pytorch/pytorch#143910]
### 📚 The doc issue

The link for the source in page torch.Tensor.backward is broken.[link](https://pytorch.org/docs/stable/generated/torch.Tensor.backward.html)

cc @svekars @brycebortree @sekyondaMeta @AlannaBurke @ezyang @albanD @gqchen @pearu @nikitaved @soulitzer @Varal7 @xmfan

**Comment by albanD** [comment]
There most likely shouldn't be any source link on this one. Or a custom link to some c++ code.
@svekars what's the right way to handle these?

**Comment by svekars** [comment]
I don't believe this is an issue anymore in the new theme: https://pytorch.org/docs/main/generated/torch.Tensor.backward.html.
